autograder/submitty_autograding_shipper.py

```diff
@@ -635,9 +635,19 @@ def grade_queue_file(config, my_name, which_machine, which_untrusted, queue_file
 
     # Try to short-circuit this job. If it's possible, then great! Clean
     # everything up and return.
-    if try_short_circuit(config, queue_file):
-        grading_cleanup(config, my_name, queue_file, grading_file)
-        return
+    try:
+        if try_short_circuit(config, queue_file):
+            grading_cleanup(config, my_name, queue_file, grading_file)
+            return
+    except Exception as e:
```

Comment:
While I'd like to see this exception be narrowed, the underlying function calls within `try_short_circuit` and `grading_cleanup` do not do a great job of narrowing themselves on possible (and also have their own `exception Exception` clauses) so fine to punt on improving this for now.